src/services/userServices.js

```diff
@@ -186,4 +186,31 @@ const storeData = (data, remove = false, config = { extend: true }) =>
  */
 const removeStoredData = () => storeData(null, true);
 
-export { checkUser, createUser, deleteUser, loginUser, logoutUser, storeData, removeStoredData };
+const setProgress = progress => {
+  window.localStorage.setItem(
+    `userProgress-${window.localStorage.getItem('currentUserName')}`,
+    JSON.stringify(progress)
+  );
+  // window.localStorage.setItem(`threadState-${thread}`, JSON.stringify(threadState));
+  return progress;
+};
+
+const getProgress = () =>
+  // const progress = {};
```

Comment:
Maybe add a TBD with why it's commented out for now, or just remove

Reply:
Acutally this should be deleted good catch